docker-compose.yml

```diff
@@ -26,8 +26,6 @@ services:
       - ./maps-config/osm-bright.tm2source/imposm_mapping.yml:/etc/imposm/mapping.yml
       - ./workspace/osm-initial-import.sh:/usr/bin/osm-initial-import
 
-    extra_hosts:
-      - "dockerhost:${DOCKER_HOST_IP}"
```

Comment:
What was this host for?  I didn't see a matching string anywhere in the repo, maybe it's a mediawiki container?